src/display/api.js

```diff
@@ -3030,6 +3041,13 @@ class WorkerTransport {
     return this.messageHandler.sendWithPromise("GetMarkInfo", null);
   }
 
+  getXFADatasets() {
+    if (typeof PDFJSDev === "undefined" || PDFJSDev.test("TESTING")) {
+      return this.messageHandler.sendWithPromise("GetXFADatasets", null);
+    }
+    return null;
+  }
+
```

Comment:
You might be able to limit this method to *only* the builds where it makes sense, by instead adding something like the following at the end of the `WorkerTransport`-constructor:
```javascript
if (typeof PDFJSDev === "undefined" || PDFJSDev.test("TESTING")) {
  Object.defineProperty(this, "getXFADatasets", {
    value: () => {
      return this.messageHandler.sendWithPromise("GetXFADatasets", null);
    },
  });
}
```